cypress/utils/utils.ts

```diff
@@ -367,17 +367,23 @@ export function removeMember(memberName: string): void {
 
 export function exists(value: string, tableSelector = appTable): void {
     // Wait for DOM to render table and sibling elements
-    cy.get(tableSelector, { timeout: 5 * SEC }).then(($tbody) => {
-        if ($tbody.text() !== "No data available") {
+    cy.get(tableSelector, { timeout: 15 * SEC }).then(($tbody) => {
+        if (
+            $tbody.text() !== "No data available" ||
+            $tbody.text() !== "No applications available"
```

Comment:
This condition will be always true in both functions `exists` and `notExists`

If the text is equal to "no data", then the first condition will be false, but the second one will be true (because the text is different from "No apps").

If the text is equal to "no apps", then the second condition will be false, but the first one will be true (because the text is different from "No data").
If the text is neither of these two messages, then the entire condition will be true